content/post/quic/article.md

```diff
@@ -0,0 +1,76 @@
+# QUIC
+
+QUIC is a new UDP based internet transport that was invented by Google in 2013 and rolled out as an experiment in Google Chrome and Google's frontend servers over the next few years. According to Google's [measurements](https://www.ietf.org/proceedings/99/slides/slides-99-maprg-the-quic-transport-protocol-design-and-internet-scale-deployment-00.pdf), on lossy connections QUIC reduces search latency (the time it takes to fully load the Google Search page) by 3.6 to 8% and the YouTube rebuffer rate (the number of times a YouTube video stops to load new data) by 15 to 18%.
+
+In 2016, the IETF formed a Working Group to specify the QUIC protocol, which is scheduled to complete the protocol specification by mid 2019.
```

Comment:
Can we add context here about why **WE** are talking about it? Aka - "In 0.4.18, go-ipfs released experimental QUIC support, and in May 2019 libp2p released 0.1.0 of the go-libp2p-quic-transport. We aim for QUIC to become the default network transport in the near future to solve many of the long standing issues with TCP - including better NAT traversal, faster connection establishment, and lower resource utilization."

Reply:
Note as well that there is a Node.js implementation in the works, and that folks can read more on that soon! (NearForm is working on a post about the implementation work which we've helped fund)